### PR TITLE
Overview and Ingest page updates

### PR DESCRIPTION
**Pull request by benskelker**
Updates the Overview and Ingest sections of the UG.

[Overview preview](https://security-docs_182.docs-preview.app.elstc.co/guide/en/security/master/es-overview.html)
[Ingest preview](https://security-docs_182.docs-preview.app.elstc.co/guide/en/security/master/ingest-data.html)

### CODE REVIEW

File: docs/es-overview.asciidoc [hunk 1]
```diff
@@ -2,150 +2,122 @@
 [chapter, role="xpack"]
 = Elastic Security overview
 
-Elastic Security combines the threat detection of SIEM and the prevention, detection, and response of Endpoint Security via an autonomous agent into a unified software solution. These analytical and protection capabilities, leveraged by the speed and extensibility of Elasticsearch, enable analysts to defend their organization from threats before damage and loss occur.
+Elastic Security combines SIEM threat detection features with endpoint
```
Comment (DonNateR) [inline docs/es-overview.asciidoc:5]: Are we still allowed to use the word SIEM?

Comment (benskelker) [reply docs/es-overview.asciidoc:5]: Yes 😄

Comment (benskelker) [reply docs/es-overview.asciidoc:5]: Unless @MikePaquette says otherwise...

File: docs/es-overview.asciidoc [hunk 2]
```diff
@@ -2,150 +2,122 @@
...
+*** *Linux/macOS*: process, network, file
+** https://www.elastic.co/integrations?solution=security[Beat modules]: {beats}
+are lightweight data shippers. Beat modules provide a way of collecting and
+parsing specific data sets from common sources, such as cloud and OS events,
+logs, and metrics. Common security-related modules are listed
+<<enable-beat-modules, here>>. 
+* The Security app in {kib} is used to manage the *Detection engine*,
+*Cases*, and *Timeline*, as well as administer hosts running Endpoint Security:
+** Detection engine - Automatically searches for suspicious host and network
+activity:
```
Comment (jmikell821) [inline docs/es-overview.asciidoc:45]: I think we can add something at the end that leads up to the following bullets (e.g., `"...Automatically searches for suspicious host and network activity via the following tools"` or something to that effect.

File: docs/es-overview.asciidoc [hunk 3]
```diff
@@ -2,150 +2,122 @@
...
-<<machine-learning, {ml} {anomaly-jobs}>> and
-<<detection-engine-overview, detection engine rules>> provide ways to
-automatically detect suspicious activity across your entire fleet of servers
-and workstations.
-
-Elastic Security requires the following {stack} components:
+Here's an overview of the flow and its components:
+
+* Data is shipped from your hosts to {es} via beat modules and the Elastic
+Agent:
```
[the remark this answers is not in the document]
Comment (benskelker) [reply docs/es-overview.asciidoc:31]: @jmikell821 changed to:
`Data is shipped from your hosts to {es} via beat modules and the Elastic
Endpoint Security agent integration:`

File: docs/es-overview.asciidoc (hunk 2 above)
Comment (benskelker) [reply docs/es-overview.asciidoc:45]: changed to: `Detection engine - Automatically searches for suspicious host and network
activity via:`

**Pull request merged**